compute/instances/api-cli/using-routed-ips.mdx

```diff
@@ -22,7 +22,7 @@ categories:
 
 <Message type="important">
   * The routed IP feature **is not compatible** with legacy Instance types using the bootscript feature.
-  * Using static network configurations during transition can lead to a loss of internet connectivity (using DHCP is advised).
+  * If you are using a manual DHCP configuration, ensure that it is compatible with a routed IP setup.
```

Comment:
Maybe add something to specifically target the _public NIC_ (we don't wan't people to change their configuration in case they are also using VPC's dhcp)